src/PolyhedralGeometry/Polyhedron/johnson.jl

```diff
@@ -0,0 +1,2427 @@
+function _johnson_solid(::Val{9})
+  NF, sr5 = quadratic_field(5)
+  NFy, y = NF["y"]
+  MF, mr = number_field(y^2 - (5 + sr5)//10, "a")
```

Comment:
Maybe all these fields and rings should be constructed with `cached=false`? Or perhaps `_johnson_solid` should have a `cached` parameter to control this?